### PR TITLE
CE-3515: Added more variations for infobox experiment

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
## Links

* https://wikia-inc.atlassian.net/browse/CE-3515

## Description

Added more variaions for infobox experiment

## Reviewers

@Wikia/community-engineering

### CODE REVIEW

File: front/main/app/components/portable-infobox-question.js [hunk 1]
```diff
@@ -101,22 +200,27 @@ export default Ember.Component.extend(
...
+						experimentWikiPage.valueToHide,
+						experimentWikiPage.valueSeparator
+					);
 				});
 			}
 
 			return experimentWikiPage;
 		}),
-		hideInfoboxField(field, value) {
+		hideInfoboxField(field, value, separator) {
```
Comment (kamilkoterba) [inline front/main/app/components/portable-infobox-question.js]: You can use default parameter
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/default_parameters

**Comment by kamilkoterba** [comment]
Besides above comment LGTM

File: front/main/app/components/portable-infobox-question.js [hunk 2]
```diff
@@ -86,7 +88,104 @@ export default Ember.Component.extend(
...
+							level: 'hard',
+							fieldToHide: 'Armor'
+						},
+						'Nikana Prime': {
+							text: 'What is the Attack Speed of Nikana Prime?',
+							level: 'hard',
+							fieldToHide: 'Attack Speed'
+						}
+					},
+					// warframe.wikia.com
```
Comment (Grunny) [inline front/main/app/components/portable-infobox-question.js]: disney.wikia.com :)

File: front/main/app/components/portable-infobox-question.js [hunk 3]
```diff
@@ -86,7 +88,104 @@ export default Ember.Component.extend(
...
+							valueToHide: 'eyes'
+						},
+						Bellwether: {
+							text: 'What color are Bellwether\'s eyes?',
+							level: 'easy',
+							fieldToHide: 'Appearance',
+							valueToHide: 'eyes'
+						},
+						'Mr. Big (Zootopia)': {
+							text: 'What color are Bellwether\'s eyes?',
```
Comment (Grunny) [inline front/main/app/components/portable-infobox-question.js]: This is a duplicate of the above question, should it be "Mr. Big's eyes"?

**Comment by kamilkoterba** [comment]
LGTM

**Pull request merged**